Android/app/src/main/java/io/github/project_travel_mate/destinations/CityFragment.java

```diff
@@ -182,7 +182,7 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
 
             mCityAdapter.updateData(mCities);
 
-            if (mSpotlightShownCount <= 3) {
+            if (mSpotlightShownCount <= 4) {
```

Comment:
why?

Reply:
My bad, this was an experimentation I was doing on trying to solve the #539 issue.